package.json

```diff
@@ -33,6 +33,7 @@
     "mutate-fs": "^2.1.1",
     "nock": "^13.2.4",
     "npm-registry-mock": "^1.3.2",
+    "rimraf": "^6.0.1",
```

Comment:
Am I missing something or is this not being used in the PR now?

Reply:
it's used in git.js for the cleanup in the daemon test (to avoid EBUSY)